libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -305,6 +310,25 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     `;
   }
 
+  protected override updated(): void {
+    this.onTabChange();
+  }
+
+  protected onTabChange(): void {
+    for (let i = 0; i < this.tabRefs.length; i++) {
+      const tab = this.tabRefs[i].value;
+      if (!tab) {
+        continue;
+      }
+      const chip = tab.querySelector('oryx-chip');
+      if (tab.selected) {
+        chip?.setAttribute('appearance', 'success');
```

Comment:
Let's update unit tests, to guarantee that when the tab is selected the oryx-chip has the right appearance